<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionDialog</class>
 <widget class="QDialog" name="ConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connexion</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label_status">
     <property name="text">
      <string>&lt;strong&gt;Connexion au périphérique en cours&lt;/strong&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="img_search">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>wait.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Recherche de l'appareil</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="img_param">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>wait.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Récupération des caractéristiques</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="img_key">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>wait.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Récupération de la clef de chiffrement</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="img_content">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>wait.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Récupération du contenu</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="button_more">
       <property name="text">
        <string>Détails ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="button_less">
       <property name="text">
        <string>Masquer ...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="text">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetIssueConnectionLinux" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="issueLinuxWhat">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Il semble que vous n'arriviez pas à vous connecter à votre appareil. Les solutions ci-dessous peuvent vous aider à résoudre ce problème.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetAskUsrName" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Entrez votre nom d'utilisateur</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="issueLinuxUsrName"/>
         </item>
         <item>
          <widget class="QToolButton" name="issueLinuxEnterUsrName">
           <property name="text">
            <string>Valider</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="issueLinuxStatus">
        <property name="text">
         <string>En attente du nom d'utilisateur ...</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetIssueDisconnectionLinux" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="issueDisconectionLinuxWhat">
        <property name="text">
         <string>Tada</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonReconnect">
     <property name="text">
      <string>Reconnexion</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_close">
     <property name="text">
      <string>Fermer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_more</sender>
   <signal>clicked()</signal>
   <receiver>button_more</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_more</sender>
   <signal>clicked()</signal>
   <receiver>button_less</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_less</sender>
   <signal>clicked()</signal>
   <receiver>button_less</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_less</sender>
   <signal>clicked()</signal>
   <receiver>button_more</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_close</sender>
   <signal>clicked()</signal>
   <receiver>ConnectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>655</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>issueLinuxUsrName</sender>
   <signal>returnPressed()</signal>
   <receiver>issueLinuxEnterUsrName</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_more</sender>
   <signal>clicked()</signal>
   <receiver>text</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_less</sender>
   <signal>clicked()</signal>
   <receiver>text</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
